### PR TITLE
Init need not return.

### PR DESCRIPTION
**Pull request by wayilau**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

I think the init function need not return the agentClassLoader, because the getDefault does this. also this funciton not used by any other places. what about your idea?

### CODE REVIEW

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java [hunk 1]
```diff
@@ -92,15 +92,14 @@ public static AgentClassLoader getDefault() {
      * @return
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java]: Remove return if you want to change it to void.

**Comment by wu-sheng** [comment]
Travis ci fails. Update anything?

**Comment by wayilau** [comment]
> Travis ci fails. Update anything?

update code without "return" .

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java (hunk 1 above)
Comment (wayilau) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java]: done!

**Pull request merged**